deny.toml

```diff
@@ -94,8 +88,6 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
 # List of explicitly allowed licenses
 # See https://spdx.org/licenses/ for list of possible licenses
 # [possible values: any SPDX 3.11 short identifier (+ optional exception)].
```

Comment:
If I see it correctly, the `licenses.allow` here was the only thing you manually changed from https://github.com/EmbarkStudios/cargo-deny/blob/0.14.4/deny.template.toml#L102-L106 (?). So I kept those manual changes.

Reply:
Yes, that's correct. I wanted to keep the allow list explicit so it can be expanded as needed.